libdino/src/service/file_manager.vala

```diff
@@ -46,6 +46,12 @@ public class FileManager : StreamInteractionModule, Object {
         return ret;
     }
 
+    // Required to allow Windows users to receive files with illegal characters in name
```

Comment:
This only applies to Windows. Use preprocessor directives: 
```vala
#if _WIN32
...
#else
...
#endif
```
The PR would be complete if there was a similar function for Linux that would replace `/` and `NUL` with `_`.
Could you add this please?